docs/UserGuide.adoc

```diff
@@ -341,7 +342,100 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-== Features (Coming in v2.0)
+== Features(Coming in v2.0)
+
+=== Add a relationship
+
+Adds a relationship between two persons. +
+Format: `addRelationship INDEXOFSOURCE INDEXOFDESTINATION DIRECTION`
+
+****
+* Add a relationship between the person with `INDEXOFSOURCE` and `INDEXOFDESTINATION`, +
+with the direction of the relationship specified.
```

Comment:
Very strange line breaks. I know for code we are not supposed to go beyond 120 characters per line (?) but for documentation the line breaks will affect the display. So please go through all the places (throughout the documentation!) where you placed such line breaks and remove them, so that they show up nicely (the text will wrap automatically, don't worry)